jekyll/_cci2_ja/datadog-integration.adoc

```diff
@@ -0,0 +1,45 @@
+---
+
```

Comment:
Does this white space cause any issues? I actually don't know, but it's on all three pages. Preferably it gets removed though.

Reply:
No it doesn't cause issues, but it's an annoying thing that seems to be done by Crowdin, I'll remove.